### PR TITLE
[ECMS-7431] upgrade ITHIT to  v2.0.1735.0

### PR DESCRIPTION
**Pull request by MrHanachoo**

### CODE REVIEW

**Comment by MrHanachoo** [comment]
Integration PR: [https://github.com/exodev/ecms/pull/193](https://github.com/exodev/ecms/pull/193)

File: webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -13,12 +13,4 @@
     </module>
   </portlet>
 
-  <module>
-    <name>ecm-webdav</name>
-    <as>ecmWebdav</as>
-    <script>
-      <path>/js/ITHitWebDAVClient.js</path>
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/WEB-INF/gatein-resources.xml]: To be sure to understand : how is now loaded this js file ?

File: webapp/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 2]
```diff
@@ -13,12 +13,4 @@
     </module>
   </portlet>
 
-  <module>
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/WEB-INF/gatein-resources.xml]: Why adding this js file via the doHeaders method instead of gatein-resources.xml ?

Comment (MrHanachoo) [inline webapp/src/main/webapp/WEB-INF/gatein-resources.xml]: @thomasdelhomenie the  ITHitWebDAVClient.js is bugging once it is minified.
Thus, I inject it as plain js with the doHeaders method.

Comment (plamarque) [inline webapp/src/main/webapp/WEB-INF/gatein-resources.xml]: it's a pity we can't disable minification per resource in gatein-resources.xml

**Pull request closed without merging**